rts/Lua/LuaUnsyncedCtrl.cpp

```diff
@@ -2167,6 +2167,12 @@ int LuaUnsyncedCtrl::SetUnitNoMinimap(lua_State* L)
  */
 int LuaUnsyncedCtrl::SetMiniMapRotation(lua_State* L)
 {
+	if (minimap == nullptr)
+		return 0;
+	
+	if (minimap->minimapCanFlip)
+		return 0;
```

Comment:
I'd put this below the `luaL_checkfloat` line so it still does a validity check.